src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -3760,6 +3760,99 @@ private static void NewMethod()
             await TestExtractMethodAsync(code, expected);
         }
 
+        // dataflow in and out can be false for symbols in unreachable code
+        // boolean indicates 
+        // dataFlowIn: false, dataFlowOut: false, alwaysAssigned: true, variableDeclared: false, readInside: true, writtenInside: false, readOutside: false, writtenOutside: true
```

Comment:
@agocke Can you confirm these are the expected values from data flow analysis here?

Reply:
@agocke these 2 cases are not for local function but for unreachable code. 

..

flow analysis not working correctly on certain language feature can result in 2 different outcome. one being infobar like this. or extract method generating broken code. not sure what known issue mentioned below on local function will result in.

Reply:
@agocke basically, it is reversed representation. data flow analysis API gives set of symbols on each cases, data flow in, out, read in/out, write in/out and etc.

I have variable for each case for a symbol that indicates the symbol exist in each set or not. if that variable is confusing you, think it as dataFlowAnalysisResult.DataFlowIn.Contains(symbol) and etc.

Reply:
LGTM